src/main/java/org/entando/kubernetes/model/bundle/processor/AbstractPageProcessor.java

```diff
@@ -127,4 +175,22 @@ protected EntandoCoreClient getEngineService() {
         return engineService;
     }
 
+    private void replaceBundleIdPlaceholder(String bundleId, PageDescriptor descriptor) {
```

Comment:
@firegloves this replaceBundleIdPlaceholder invokes  ProcessorHelper.applyBundleIdPlaceholderReplacement
which in turns invokes the internal ProcessorHelper.replaceBundleIdPlaceholder

this can be confusing and in particular the names doesn't seems to represent very well the intent